package.json

```diff
@@ -16,9 +17,9 @@
     "test": "pnpm -r test",
     "code:checks": "pnpm prettier:check && pnpm lint:check && pnpm ts:check",
     "code:fixes": "pnpm prettier:fix && pnpm lint:fix",
-    "postinstall": "pnpm --filter cloudflare build",
```

Comment:
There is not much value in running a build in the postinstall.
Anyone who is developing on this repo would need to run the build a lot more often.

Reply:
Actually this is needed so the cloudflare binary is available to other packages in the monorepo.